pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -320,6 +321,11 @@ protected BrokerResponse handleRequest(long requestId, String query, @Nullable S
         // Compile the request into PinotQuery
         compilationStartTimeNs = System.nanoTime();
         pinotQuery = CalciteSqlParser.compileToPinotQuery(sqlNodeAndOptions);
+        if (pinotQuery.getDataSource() != null && pinotQuery.getDataSource().getTableName() != null) {
+          String tableName = getActualTableName(DatabaseUtils.translateTableName(
+              pinotQuery.getDataSource().getTableName(), httpHeaders), _tableCache);
+        pinotQuery.getDataSource().setTableName(tableName);
+        }
```

Comment:
Revert this? Table name parsing is handled below

Reply:
Yes, earlier I was not sure how the `GapfillUtils.stripGapfill(pinotQuery)` worked but as it was reassigning the PinotQuery and original PinotQuery was also being used later in the method by `getEmptyBrokerOnlyResponse`. I played safe and added translation for both of them. Now that you pointed it out, it seems that translation on just the reassigned PinotQuery should be sufficient. Will revert the first translation.
Thanks for catching it!